pages/robots/rbts-intern-orientation.md

```diff
@@ -59,8 +59,8 @@ You should check if the following conditions are met:
 * [x] include issue number in pull request title
 * [x] no unneeded files/lines change in pull request
```

Comment:
I think we should rewrite this to: "don't include unnecessary files/lines change in the pull request"